dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java

```diff
@@ -610,263 +593,248 @@
     public static final String REFER_KEY = "refer";
 
     /**
-     * callback inst id
+     * To decide whether to make connection when the client is created
      */
-    public static final String CALLBACK_SERVICE_KEY = "callback.service.instid";
+    public static final String LAZY_CONNECT_KEY = "lazy";
 
-    /**
-     * The limit of callback service instances for one interface on every client
-     */
-    public static final String CALLBACK_INSTANCES_LIMIT_KEY = "callbacks";
+    public static final String DUBBO_PROTOCOL = DUBBO;
 
-    /**
-     * The default limit number for callback service instances
-     *
-     * @see #CALLBACK_INSTANCES_LIMIT_KEY
-     */
-    public static final int DEFAULT_CALLBACK_INSTANCES = 1;
+    public static final String ZOOKEEPER_PROTOCOL = "zookeeper";
 
-    public static final String CALLBACK_SERVICE_PROXY_KEY = "callback.service.proxy";
+    // FIXME: is this still useful?
+    public static final String SHUTDOWN_TIMEOUT_KEY = "shutdown.timeout";
 
-    public static final String IS_CALLBACK_SERVICE = "is_callback_service";
+    public static final int DEFAULT_SHUTDOWN_TIMEOUT = 1000 * 60 * 15;
```

Comment:
This is not used, it is recommended to delete

Reply:
let's do it later.